src/api/lib/tasks/sphinx.rake

```diff
@@ -8,8 +8,31 @@ namespace :sphinx do
       Rake::Task['ts:start'].invoke
     end
   end
+
+  desc 'Start the sphinx daemon for the development environment'
+  task start_for_development: :environment do
+    if index_to_build?
+      Rake::Task['ts:clear'].invoke
+      Rake::Task['ts:configure'].invoke
+      t = Thread.new do
+        counter = 0
```

Comment:
:thought_balloon:  What about renaming this to `retries`?

Reply:
Done.